Dockerfile

```diff
@@ -62,13 +62,16 @@ RUN mkdir /opt/taudem && \
 ENV PATH /opt/taudem/:$PATH
 
 # Download and build RWD
+# The SHA in the following URL should be kept in sync with the latest commit on
+# the develop branch. If we simply used develop.tar.gz, this command would
+# not be executed (due to the cache) and we would not get the latest commits.
```

Comment:
We'd like to use the cache for everything we can, but avoid it for this line and everything below it.